src/main/java/com/joss/conductor/mobile/ConductorConfig.java

```diff
@@ -500,9 +507,20 @@ public String getFullAppPath() {
         if (appFile == null) {
             return null;
         }
-
+        File dir = new File(appFile);
         Path path = Paths.get(appFile);
-        if (appFile.startsWith("sauce-storage:") || path.isAbsolute()) {
+        if (dir.isDirectory()) {
+            try {
+                String expectedFileExtension = platformName == Platform.ANDROID ? ".apk" : ".app";
+                File[] files = dir.listFiles((dir1, name) -> name.toLowerCase().endsWith(expectedFileExtension));
+                if (files != null) {
+                    File fileFoundMatchingExtension = files[0];
+                    return Paths.get(System.getProperty("user.dir"), fileFoundMatchingExtension.toString()).normalize().toString();
+                }
+            } catch (Exception e) {
+                Logger.error(e, "Error attempting to find {} from relative path", appFile);
```

Comment:
Does this print out the error message from the exception?

Reply:
Yes, TinyLog should do this by default

See:
![image](https://user-images.githubusercontent.com/22263679/62556124-3d75e680-b842-11e9-821e-f7191a120212.png)
From: https://tinylog.org/logging